### ISSUE TITLE
Indeed offset doesn't work in python

### ISSUE THREAD

**Issue by alisalamin** [Bunsly/JobSpy#213]
No amount of fiddling with the offset value seems to provide results past page 1.

**Comment by alisalamin** [comment]
This will fail :


```python
from jobspy import scrape_jobs
import pandas as pd


def test_indeed():
    result = scrape_jobs(
        site_name="indeed",
        search_term="engineer",
        results_wanted=5,
        offset=1 # <----
    )
    assert (
        isinstance(result, pd.DataFrame) and len(result) == 5
    ), "Result should be a non-empty DataFrame"


```

**Comment by cullenwatson** [comment]
ok fixed in 1.1.72

**Comment by alisalamin** [comment]
Thanks again for the quick fix!
